connector_magento/models/product/common.py

```diff
@@ -108,12 +108,13 @@ def recompute_magento_qty(self):
         informations for each product.
         """
         # group products by backend
-        backends = defaultdict(self.browse)
+        backends = defaultdict(set)
```

Comment:
@guewen  If you want to squeeze some more perfs out of this, make this `defaultdict(list)` and on line 117 below use `self.browse(set(product_ids))`